packages/transport/http.go

```diff
@@ -39,35 +70,259 @@ func marshalMsgs(r *http.Request, msgs []set.Token) ([]byte, error) {
 	return arr, nil
 }
```

Comment:
Is MasrshalMsgs function necessary? Since I get the msg from db as json. Maybe we can use json.marshal directly in handle function.